go/libraries/doltcore/sqle/views_test.go

```diff
@@ -30,7 +30,6 @@ import (
 // Not an exhaustive test of views -- we rely on bats tests for end-to-end verification.
 func TestViews(t *testing.T) {
 	dEnv := dtestutils.CreateTestEnv()
-	CreateTestDatabase(dEnv, t)
```

Comment:
?

Reply:
`CreateTestDatabase` creates the Simpson's tables which have conflicting tags and cause issues in `ExecuteSql`
The test doesn't rely on those tables, so I dropped it.